aero.py

```diff
@@ -445,23 +455,54 @@ def run_in_emulator(args, iso_path):
     if is_kvm_available and not args.disable_kvm:
         print("Running with KVM acceleration enabled")
 
-        qemu_args += ['-enable-kvm', '-cpu',
-                      'host,+la57' if args.la57 else 'host']
+        if platform.system() == 'Darwin':
+            qemu_args += ['-accel', 'hvf']
+        else:
+            qemu_args += ['-enable-kvm']
+        qemu_args += ['-cpu', 'host,+la57' if args.la57 else 'host']
     else:
         qemu_args += ["-cpu", "qemu64,+la57" if args.la57 else "qemu64"]
 
     run_command(['qemu-system-x86_64', *qemu_args])
 
 
+def get_sysctl(name: str) -> str:
+    """
+    Shell out to sysctl(1)
+
+    Returns the value as a string.
+    Non-leaf nodes will return the value for each sub-node separated by newline characters.
+    """
+    status, stdout, stderr = run_command(["sysctl", "-n", name],
+                                         stdout=subprocess.PIPE,
+                                         stderr=subprocess.PIPE)
+    if status != 0:
+        print("`sysctl` failed:")
```

Comment:
Maybe make this:

```python
print("`sysctl` failed: ", end="")
print(stderr.decode())
```

This way the full error can be on the same line ;^)

Reply:
on it